pkgs/applications/misc/ubpm/default.nix

```
@@ -0,0 +1,51 @@
+{ stdenv, lib, fetchFromGitea, qmake, qttools, qtbase, qtserialport
+, qtconnectivity, qtcharts, qttranslations, wrapQtAppsHook }:
+
+stdenv.mkDerivation rec {
+  pname = "ubpm";
+  version = "1.7.3";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "LazyT";
+    repo = pname;
```

```suggestion
    repo = "ubpm";
```

See https://github.com/nix-community/nixpkgs-lint/issues/21#issuecomment-1618243610